aya/src/maps/perf/perf_event_array.rs

```diff
@@ -175,7 +176,7 @@ impl<T: Borrow<MapData>> PerfEventArray<T> {
     }
 }
 
-impl<T: BorrowMut<MapData>> PerfEventArray<T> {
+impl<T: Borrow<MapData>> PerfEventArray<T> {
```

Comment:
Why change this? I think we want to keep open() only for BorrowMut. I think
`pin` should go in the impl above?

Reply:
Good catch can't remember why I did this thanks!